xknx/devices/climate.py

```diff
@@ -228,16 +207,19 @@ def initialized_for_setpoint_shift_calculations(self):
     def temperature_step(self):
         """Return smallest possible temperature step."""
         if self._setpoint_shift.initialized:
-            return self._setpoint_shift.setpoint_shift_step
-        return DEFAULT_TEMPERATUE_STEP
+            return self.setpoint_shift_step
```

Comment:
I think in SetpointShiftMode.DPT9002 we should use a default of `0.1` not `0.5`
What is the default on MDT actors?

Reply:
The default of the "Glastaster II" is 0.5.

Reply:
The heating actuator doesn't allow configuration of step when using 2byte...

Reply:
This is correct. Does it make more sense to use `DEFAULT_TEMPERATURE_STEP` for this mode? I think `DEFAULT_TEMPERATURE_STEP` and `DEFAULT_SETPOINT_SHIFT_STEP` are kinda redundant.

Reply:
Yes your right, but 0.1 is required for direct (and in my opinion dpt9 sps). So we would have to change default value for sps dpt6 and this would break some default configuration.

Reply:
I do not understand what you mean with "required" for direct and dpt9 sps. Both methods can send whatever temperature/offset they want, if I understood correctly.

Reply:
your right, its not really required... HA thermostat card uses this for their sliders. So when one has set eg. 21.3 from some other knx device it is awkward to have a slider in HA that can be set to 20.3, 20.8, 21.8, 22.3 ... therefore I would prefer 0.1 as a default.

Reply:
But the default of 0.1 will break the DPT6 SPS method. So what are we gonna do? Keep both and use `DEFAULT_TEMPERATURE_STEP` for DPT9 SPS and direct?

Reply:
We can keep both until we introduce the next breaking change for Climate and clean this up then.

Reply:
So `DEFAULT_TEMPERATURE_STEP` for DPT9 SPS and direct, the other one for DPT7 SPS?

Reply:
Then go for it, its configurable anyway. You could add a note to the documentation.